ui/src/app/components/cases/test-case-details.component.ts

```diff
@@ -65,7 +65,7 @@ export class TestCaseDetailsComponent extends BaseComponent implements OnInit {
     super(authGuard, notificationsService, translate,toastrService);
   }
 
-  get isGroup() {
+  get checkCaseType() {
```

Comment:
**check** prefix refers to boolean value, but it actually returns a string, so plz have a better naming

Reply:
i changed the value from isGroup to checkCaseType. it returns string